### ISSUE TITLE
[Ic-Button] - tooltips do not work on icon button variants

### ISSUE THREAD

**Issue by ad9242** [mi6/ic-ui-kit#2299]
## Summary of the bug 
When using a IcButton with `variant=icon` a tooltip is displayed when an aria-label is set. However, this does not work for the newly added icon- variants eg "icon-primary", "icon-secondary" etc

## 🪜 How to reproduce 
Tell us the steps to reproduce the problem: 
1. Go to [Storybook](https://mi6.github.io/ic-ui-kit/branches/develop/web-components/?path=/story/web-components-button--icon-variants)
2. Hover over the first button shown
3. No tooltip is visible - it should have one saying "refresh"

## 🧐 Expected behaviour 
All icon variants of buttons should support tooltips

## 📝 Acceptance Criteria  
If relevant, describe in full detail the different interactions and edge cases that the component or patterns needs to fulfil. 

*Given* an icon variant of button
*When*  an arai-label is set
*Then* a tooltip should be displayed when the button is focused or hovered

## Additional info 
Tell us anything else useful to help us fix or understand the problem.
